apps/condo/domains/acquiring/schema/MultiPayment.js

```diff
@@ -153,7 +155,16 @@ const MultiPayment = new GQLListSchema('MultiPayment', {
         integration: ACQUIRING_INTEGRATION_FIELD,
     },
     hooks: {
-        // TODO (savelevMatthew): Create proper validations
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // TODO (savelevMatthew): Write more complex validations
```

Comment:
please add ticket number for TODO here

Reply:
Sure, but give me strict format for ticket + nickname todos

Reply:
Change to ticket-number....but we definitely need to think about named TODOs as soon as we can